src/main/target/CC3D/target.h

```diff
@@ -44,7 +44,7 @@
 
 #define USABLE_TIMER_CHANNEL_COUNT 12
 
-#define DEBUG_MPU_DATA_READY_INTERRUPT
+//#define DEBUG_MPU_DATA_READY_INTERRUPT
```

Comment:
Was this intentional? The change is only made on one target and is a comment instead of a removal.